Utilities/build-script-helper.py

```diff
@@ -148,6 +154,8 @@ def add_common_args(parser):
     parser.add_argument('--sanitize', action='append', help='build using the given sanitizer(s) (address|thread|undefined)')
     parser.add_argument('--sanitize-all', action='store_true', help='build using every available sanitizer in sub-directories of build path')
     parser.add_argument('--verbose', '-v', action='store_true', help='enable verbose output')
+    parser.add_argument('--cross-compile-host', help='cross-compile for another host instead')
```

Comment:
Unlike other `build-script` products that added this build flag, [I made it singular](https://github.com/apple/swift/pull/36917#issue-615550282), as we only ever pass one cross-compilation host at a time.